docker/monitoring/docker-compose.consumers.monitoring.yml

```diff
@@ -1,5 +1,5 @@
 ---
-version: '3.5'
```

Comment:
why was this needed to bump?

Reply:
to make it compatible with other docker-compose file under [datahub](https://github.com/datahub-project/datahub)/[docker](https://github.com/datahub-project/datahub/tree/master/docker)/docker-compose-without-neo4j.yml